icons/nasa.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>NASA icon</title><path d="M4.344 13.598c.075.281.195.39.407.39.22 0 .335-.132.335-.39V8.804h1.379v4.794c0 .675-.088.968-.43 1.31-.247.248-.703.439-1.278.439-.464 0-.909-.154-1.192-.438-.249-.25-.386-.505-.599-1.311l-.846-3.196c-.074-.281-.194-.39-.406-.39-.22 0-.336.132-.336.39v4.794H0v-4.794c0-.675.088-.968.43-1.31.247-.248.703-.439 1.278-.439.464 0 .909.154 1.192.438.249.25.385.505.599 1.311zM22.575 15.196l-1.591-4.98a.415.415 0 00-.06-.132.226.226 0 00-.186-.082.226.226 0 00-.185.082.414.414 0 00-.06.132l-1.591 4.98h-1.425l1.739-5.44c.09-.283.22-.524.384-.684.282-.275.614-.419 1.138-.419.525 0 .857.144 1.139.42.164.16.294.4.384.683L24 15.196h-1.425zM15.531 15.196c.903 0 1.344-.192 1.692-.538.385-.383.569-.802.569-1.427 0-.553-.202-1.064-.51-1.37-.403-.4-.903-.527-1.719-.527h-1.142c-.436 0-.61-.053-.748-.188-.094-.093-.139-.23-.139-.393 0-.168.04-.334.156-.448.103-.1.243-.147.511-.147h3.301V8.804h-3.049c-.903 0-1.343.192-1.691.538-.385.383-.57.802-.57 1.427 0 .553.203 1.064.51 1.37.404.4.904.527 1.72.527h1.141c.437 0 .61.053.748.188.095.093.14.23.14.393 0 .169-.041.335-.157.448-.102.1-.242.147-.51.147h-3.405l-1.306-4.086c-.09-.283-.22-.524-.384-.684-.282-.275-.615-.419-1.139-.419s-.857.144-1.138.42c-.165.16-.294.4-.385.683l-1.738 5.44h1.424l1.592-4.98a.415.415 0 01.06-.132.226.226 0 01.185-.082c.082 0 .142.028.186.082a.413.413 0 01.06.132l1.591 4.98h4.144z"/></svg>
```

Comment:
Other than a load of points in both `A` shapes, this looks spot on. 💯
@adamalston could you have a look at reducing these points down - but no worries if it distorts the icon.

![image](https://user-images.githubusercontent.com/1830867/105719879-b2e3e400-5f1a-11eb-9a75-0165eedd2a87.png)

Reply:
I opened the svg in an editor and I do not see any extraneous points. Can you provide more insight on how to find/reproduce the errors?

Reply:
@adamalston yep, so I've opened the icon in Figma, and double clicked on the vector to show the points. A lot of vector tools have options to simplify paths - but not to worry if you're unable to access it. As I say the icon is spot on, just those points which would be a minor optimization. 💪

Reply:
When I simplify the path in the editor, the shape of the letters is distorted. 

If you would like to submit a change, I'm fine with that. Otherwise, I don't know how to fix the extraneous points at the moment.

Reply:
No worries - I'm not able to edit the points in Inkscape, and the icon just won't open in Illustrator! 😅
Happy to merge this as is, unless another @simple-icons/maintainers (like @fbernhart 🧙🏻‍♂️) is able to optimize further. Will wait until later today, just in case they're able to get them.

Reply:
>  the icon just won't open in Illustrator!

Chiming in here just to say something about this. This is likely due to the icon being optimized with [SVGOMG](https://jakearchibald.github.io/svgomg/). It does some optimization that Illustrator doesn't like. You can fix this by reoptimizing with our [`.svgo.yml` configuration](https://github.com/simple-icons/simple-icons/blob/3166f14e71bcbdda1a61fc2a6b4513b3ab493982/.svgo.yml). (For more details see https://github.com/simple-icons/simple-icons/issues/2061 and related thread :slightly_smiling_face:)